Update cloud-seats-and-users.md (#3959)
fishtown-analytics/docs.getdbt.com: website/docs/docs/cloud/manage-access/cloud-seats-and-users.md

| Use [Source Freshness](/docs/deploy/source-freshness) | ✅ | ✅ | ❌ |
| Use [Docs](/docs/collaborate/build-and-view-your-docs) | ✅ | ✅ | ❌ |
| Receive [Job notifications](/docs/deploy/job-notifications) |  ✅ |  ✅ |  ✅ | 

> *Available on Enterprise and Team plans only and doesn't count toward seat usage. Please note, IT seats is limited to 1 seat for a Team plan account.

Reviewer: ```suggestion
*Available on Enterprise and Team plans only and doesn't count toward seat usage. Please note, IT seats are limited to 1 seat per Team plan account.
```
